Apps/frontend/src/components/WaveControls.svelte

```diff
@@ -7,10 +7,48 @@
   import { timeSweep } from "../stores.js";
   import TimeSweepSlider from "./TimeSweepSlider.svelte";
   import ThicknessSwitch from "./ThicknessSwitch.svelte";
+  import DistributeOffsetButton from "./DistributeOffsetButton.svelte";
+  import startStopLine from "./Waves.svelte";
   export let waveElement;
   export let isEnabled;
   export let indicatorElement;
   let btnOnOff;
+
+  let activeChannels = [];
+  let activeChannelCounter = NUM_CHANNELS;
+
+  let distributed = false;
+
+  const setActiveChannels = () => {
```

Comment:
This is not `setActiveChannels`, it is much rather `disableAllChannels`, isn't it?

Reply:
Got removed anyways